src/discord.ts

```diff
@@ -0,0 +1,73 @@
+import { Client } from "discord.js";
+import { onSelector } from "./store";
+import { createSelector } from "@reduxjs/toolkit";
+import { getLinkedAccounts } from "./store/link";
+import { getParticipants, getHasSeenStart, getCallIsActive } from "./store/zoom";
+import { mapToArray } from "./helpers";
+import { DISCORD_ACTIVE_ROLE } from "./config";
+
+
+const getActiveDiscordUsers = createSelector(
+	getLinkedAccounts, 
+	getParticipants, 
+	(linkedAccounts, participants) => {
+		return participants.map(({ id }) => linkedAccounts.byId[id].discord)
+});
+
+const getZoomInfo = createSelector(
+  getCallIsActive,
+  getHasSeenStart,
+  getParticipants,
+  (active, hasSeenStart, participants) => ({
+    active,
+    hasSeenStart,
+    participants,
+  })
+)
+
+// keeps discord in sync with the server side state
+export const syncDiscordStatus = (discord: Client) => {
```

Comment:
Nit: for clarity, can we have this be called `subscribeToDiscordStatus` or something like that?

Reply:
I agree that the name is unclear, but subscribe suggests that we are subscribing to some state/events (which we technically are) when the point of the function is to push state changes to discord. I'm not a good way to describe what it does